docs/content/en/docs/topics/accessing-services/stargate/_index.md

```diff
@@ -6,39 +6,141 @@ description: |
-endpoints. These endpoints may be scaled horizontally independently of data
-layer scaling needs. This guide covers accessing the various API endpoints
+in front of the Apache Cassandra cluster providing CQL, REST, GraphQL, and document based API
+endpoints. These endpoints may be scaled independently of the data
+layer as necessary. This guide covers accessing the various API endpoints
 provided by Stargate.
 
+While this document will help get you up and going quickly with Stargate, more detailed 
+information about using Stargate can be found in the Stargate 
+[docs](https://stargate.io/docs/stargate/1.0/quickstart/quickstart.html). 
+
 ## Tools
 
 * HTTP client (cURL, Postman, etc.)
-* GraphQL client
-* CQL client (application, cqlsh, etc.)
+* Web Browser
 
 ## Prerequisites
 
-1. Kubernetes cluster with the following elements deployed:
-   * [K8ssandra Operators]({{< ref "getting-started#install-k8ssandra" >}}) Helm
-     Chart
-   * [K8ssandra Cluster]({{< ref "getting-started#install-k8ssandra" >}}) Helm
-     Chart
-   * [Ingress Controller]({{< ref "ingress" >}})
-1. DNS name configured for the Stargate interface, referred to as _STARGATE
-   DOMAIN_ below.
-1. Port number for the Stargate CQL interface
+1. [K8ssandra Cluster]({{< ref "getting-started#install-k8ssandra" >}})
+1. [Ingress]({{< ref "ingress" >}}) configured to expose each of the Stargate services (Auth, REST, GraphQL)
+1. DNS names configured for the exposed Stargate services, referred to as `STARGATE_AUTH_DOMAIN`, `STARGATE_REST_DOMAIN`, and `STARGATE_GRAPHQL_DOMAIN` below.
+
+## Access Auth API
+
+Before accessing any of the provided Stargate data APIs, an auth token must be generated and provided
+to subsequent data API requests.  This token generation process is done via the auth API.
+
+The default port exposed by Stargate for the auth API is `8081`, these examples will assume that is the
+port exposed by the cluster ingress configuration for access.
+
+The authorization API can be accessed at: [http://STARGATE_AUTH_DOMAIN/v1/auth](http://STARGATE_AUTH_DOMAIN/v1/auth)
+
+Detailed information about the Stargate auth API can be found in the Stargate [docs](https://stargate.io/docs/stargate/1.0/developers-guide/auth.html).
+
+### Extracting Cassandra username/password Secrets
+
+The auth API requires the Cassandra username and password to be provided to it.  Those values can be 
+extracted from the K8ssandra cluster through the following commands (replace `k8ssandra-cluster` with the
+name configured for your running cluster).
+
+Extract and decode the username secret:
+
+```
+kubectl get secret k8ssandra-cluster-superuser -o jsonpath="{.data.username}" | base64 --decode
+```
+
+Extract and decode the password secret:
+
+```
+kubectl get secret k8ssandra-cluster-superuser -o jsonpath="{.data.password}" | base64 --decode
+```
+
+### Generating Auth Tokens
+
+With the secrets extract and decoded, a request to generate a token can now be sent to the Stargate auth API.
```

Comment:
Grammar nitpick:

```suggestion
With the secrets extracted and decoded, a request to generate a token can now be sent to the Stargate auth API.
```

Also, the second half of the sentence is passive voice. I'd be tempted to rephrase as something like "we are now ready to request a token from the Stargate auth API," but the use of "we" might go against style guidelines. Do we have any such guidelines?

Reply:
Perhaps:
```suggestion
Use the extracted and decoded secrets to request a token from the Stargate auth API.
```